compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -33,6 +33,12 @@ import scala.collection.mutable
  */
 case class Completion(label: String, description: String, symbols: List[Symbol])
 
+/** Mode of code completion specifying which members it can see: all, explicit or implied */
+enum ImportMode { case All, Explicit, Implied }
```

Comment:
Can't use a dotty enum in the compiler until we drop the 2.12 CI: https://github.com/lampepfl/dotty/blob/9a2d2d1f0dac8618832dda202cf1946c30991001/.drone.yml#L60 :)

Reply:
Is there any issue blocking that? Or can I just drop it?

Reply:
We wanted to wait a bit and see if any blocker forced us to revert to 2.12. Let's discuss on Monday if it's time to drop 2.12 completely.

Reply:
Yes, this new commit is not pretty